src/common/pico_stdlib/include/pico/stdlib.h

```diff
@@ -56,6 +56,10 @@ extern "C" {
 #include "pico/stdio_semihosting.h"
 #endif
 
+// PICO_CONFIG: PICO_DEFAULT_LED_PIN, Optionally defined a pin that drives a regular LED on the board, default=undefined, group=pico_stdlib
```

Comment:
The comments on #191 kinda imply we might also want an optional boolean `PICO_DEFAULT_LED_PIN_ACTIVE_HIGH` config-setting? (defaulting to 1, can be set to 0 for those boards with an active-low LED)